File: 프로그래머스/3/42884. 단속카메라/단속카메라.py
def solution(routes):
    answer = 0
    routes.sort(key=lambda x:x[1])  # 진출 지점 기준 오름차순
    current = -300001
    for route in routes:
        if route[0] > current:
            answer +=1
            current = route[1]
    return answer

File: 프로그래머스/3/42884. 단속카메라/단속카메라.js
function solution(routes) {
    var answer = 0;
    routes.sort((a,b)=>a[1]-b[1])
    current = -30001
    console.log(routes)
    for (let route of routes){
        if(route[0]>current){
            current = route[1];
            answer ++;
        }
    }
    return answer;
}